### PR TITLE
CO-146: remove play dependencies from sbt-cake

### PR DESCRIPTION
**Pull request by Slakah**
https://cakesolutions.jira.com/browse/CO-146

Used by: cakesolutions/play-scala-template.g8#34

- [x] remove play dependencies from sbt-cake
- [x] remove play web dependencies
- [x] remove janino
- [x] remove akka-http-play-json dependency

### CODE REVIEW

File: project/project/CakePlatformDependencies.scala [hunk 1]
```diff
@@ -113,7 +113,6 @@ object CakePlatformDependencies {
     val packager: ModuleID =
```
Comment (carlpulley) [inline project/project/CakePlatformDependencies.scala:108]: We also want to remove a number of the other Play related dependencies here - e.g. webjars grouping, janino(?), playJson, sbt-js-engine and sbt-web.

**Review by carlpulley — CHANGES_REQUESTED** [review]

File: project/project/CakePlatformDependencies.scala (hunk 1 above)
Comment (Slakah) [reply project/project/CakePlatformDependencies.scala:108]: playJson is currently only used for `akka-http` and I think janino is mainly used for colouring log output (correct me if I'm wrong).

I will remove the others.

Comment (carlpulley) [reply project/project/CakePlatformDependencies.scala:108]: The akka-http template is using spray-json, so it sounds like we can drop the playJson dependency (assuming that the play template has no need of it).

janino is currently only used in play template for colouring. Again, akka-http is not using this, so perhaps we should move this to the play template?

**Review by carlpulley — APPROVED** [review]

**Comment by carlpulley** [comment]
We need to just check that this PR builds successfully in Jenkins before we can merge it.

**Comment by ferhtaydn** [comment]
@Slakah I think you need to fix sbt-scripted tests, in `sbt-test/sbt-cake/simple/build.sbt` that has `enablePlay` keyword even you removed from sbt-cake, to make jenkins happy. Then we can merge that. /cc @carlpulley

**Comment by Slakah** [comment]
@ferhtaydn, removed in 8d51d0778b4fe196ae9382e9245ef0a8cfbd9468, did a quick sanity grep as well `grep -r enablePlay .`.

**Pull request merged**